example/basic_example.py

```diff
@@ -26,16 +31,16 @@
 
 
 # Callback to gather cpu usage
-def get_cpu_usage_callback(observer):
+def get_cpu_usage_callback():
     for (number, percent) in enumerate(psutil.cpu_percent(percpu=True)):
-        labels = {"cpu_number": str(number)}
-        observer.observe(percent, labels)
+        attributes = {"cpu_number": str(number)}
+        yield Measurement(number, attributes)
```

Comment:
This yields the CPU number as the value, instead of the percentage

Reply:
Yes, that's a bug. :grimacing:  Thanks for catching that one! :slightly_smiling_face: 
Fixed in [e2f3a29](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/22/commits/e2f3a29cfbef67b8abee561aa685b55d1b7a2d1c).